core/api/src/services/bria/index.ts

```diff
@@ -258,6 +261,10 @@ export const OnChainService = (): IOnChainService => {
     }
   }
 
+  const getPayoutSpeeds = (): PayoutSpeedRes[] => {
+    return briaConfig.payoutQueues
```

Comment:
probably this will cause problems in staging/prod, please return the payouts that match queues in bria (like an inner join :) )

Reply:
*btw, there is a description in bria, please use it if the config description is empty